en/docs/_ci/travis.md

```diff
@@ -14,5 +24,14 @@ cache:
 
 {% include_relative _ci/deb-specific-version.md %}
 
-> Let Travis CI know if you want Yarn to be
-> [installed by default](https://github.com/travis-ci/travis-ci/issues/6720).
+## container-based builds
+
+Container-based builds do not have the `sudo` privilege, so they must rely on other means to install.
```

Comment:
@BanzaiMan - Why so? A Docker container should be reasonably contained such that even superuser permissions can't affect anything other than the container itself. CircleCI let people use `sudo` and also SSH into their containers 😄